src/components/segmentedControl/segmentedControl.svelte

```diff
@@ -122,8 +122,7 @@
     --leo-control-item-font: var(--leo-font-components-button-default);
 
     display: flex;
-    min-width: max-content;
-    max-width: var(--leo-segmented-control-max-w, none);
+    width: var(--leo-segmented-control-width, fit-content);
```

Comment:
So, just to make things much more tricky for you could you make this work with `:host`?

Maybe something like

```css

:host {
    width: var(--leo-segmented-control-width, fit-content);

    & .leo-segmented-control {
        width: 100%;
    }
}
```

and test it works in the plain-html examples